src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.json

```diff
@@ -9,7 +9,7 @@
         "Microsoft.Build.Utilities.Core": "0.1.0-preview-00024-160610",
         "Microsoft.Build.Targets": "0.1.0-preview-00024-160610",
         "Microsoft.Build": "0.1.0-preview-00024-160610",
-        "Microsoft.Net.Compilers.netcore": "1.3.0-beta1-20160429-01",
+        "Microsoft.Net.Compilers.netcore": "2.0.0-beta3",
```

Comment:
I really don't like seeing this version copied multiple places :( Do we have any strategies to try and avoid this duplication?